functions/npm.fish

```diff
@@ -1,3 +1,3 @@
-function npm -d "javascript package manager"
+function npm -d "javascript package manager" -c npm -w npm
```

Comment:
>  didn't you read the note I posted above?

Ah, yeah, I missed that, sorry.

Okay, this may seem to work when there is already an existing function that can be wrapped. If you test this with a function that does not exist already:

```fish
function hi -c hi
   echo hi
end
```

No `hi` function is created.